src/state/operator/selectors.js

```diff
@@ -92,27 +143,18 @@ export const selectSocketIdentity = ( { operators: { sockets, identities } }, so
 	get( sockets, socket.id )
 )
 export const selectUser = ( { operators: { identities } }, userId ) => get( identities, userId )
-export const selectTotalCapacity = ( locale, groups, state ) => compose(
-	reduce( ( { load: totalLoad, capacity: totalCapacity }, { id, status, online } ) =>
-		ifElse(
-			whereEq( { status: STATUS_AVAILABLE, online: true } ),
-			() => {
-				const { load, capacity, active } = getLocaleMembership( locale, id, state )
-				if ( ! active || ! isMemberOfGroups( id, groups ) ) {
```

Comment:
Good spotting, but this shouldn't be a problem.

The logic was duplicated in `getAvailableOperators` above: https://github.com/Automattic/happychat-service/pull/128/files#diff-01c5d5daccb55a7449faef967bdfcbf7R124

Rather than keeping the duplicate logic, I've just used `getAvailableOperators` and then reducing over them instead, so that case is covered :)